docs/development/style-guide.rst

```diff
@@ -208,49 +229,101 @@ Example
 “simply,” “just,” or “straightforward.” Avoid extraneous phrases like, “we just
 have to do one more thing.”
 
+Avoid words or phrasing that create worry in the mind of the reader, such as an
+overuse of "Warnings." Instead, use positive language that establishes
+confidence in the skills being learned, such as "As a best practice..." or "One
+recommended way to..." or "An important note to remember is..."
+
 Documentation vs. Tutorials vs. Guides
 --------------------------------------
 
 Documentation
-~~~~~~~~~~~~~
+^^^^^^^^^^^^^
 Tone: academic and slightly more formal.
 
-* Use up style capitalization in section headings.
+* Use title case capitalization in section headings.
 * Do not use contractions.
 
 Tutorials
-~~~~~~~~~
+^^^^^^^^^
 Tone: academic but less formal and more friendly.
 
-* Use up style capitalization in introductory/template headings, switch to down
-  style capitalization for learning/example section headings.
+* Use title case capitalization in introductory/template headings, switch to
+  sentence case capitalization for learning/example section headings.
 * Section headings should use the imperative mood to form a command or request
   (e.g., “Download the data”).
 * Contractions can be used as long as the tense is clear.
 
 Guides
-~~~~~~
+^^^^^^
 Tone: academic but less formal and more friendly.
 
-* Use up style capitalization in introductory/template headings, switch to down
-  style capitalization for learning/example section headings.
+* Use title case capitalization in introductory/template headings, switch to
+  sentence case capitalization for learning/example section headings.
 * Contractions can be used as long as the tense is clear.
+
+Formatting Guidelines
+=====================
+
+Astropy documentation is written in reStructuredText using the Sphinx
+documentation generator. When formatting the different sections of your
+documentation files, please follow these guidelines to maintain consistency in
+section heading hierarchy across Astropy's RST files.
+
+Section headings in reStructuredText files are created by underlining (and
+optionally overlining) the section title with a punctuation character at least
+as long as the text.
+
+Examples
+--------
+
+::
+  *************************
+  This is a Chapter Heading
+  *************************
+
+::
+  This is a Section Heading
+  =========================
+
+Although there are no formally assigned characters to create heading level
+hierarchy, as the hierarchy rendering is determined from the succession of
+headings, here is a suggested convention to follow when formatting Astropy
+documentation files:
+
+# with overline, for parts
+* with overline, for chapters
+=, for sections
+-, for subsections
+^, for subsubsections
+", for paragraphs
+
+These guidelines follow Python's recommendation in the `7.3.6. Sections
+<https://devguide.python.org/documenting/#sections>`_ part of its style guide.
```

Comment:
The authoritative source is actually at  https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections .

Reply:
Nice catch, I'll switch the links!